test/resources/router/partial/partial.js

```diff
@@ -25,6 +25,20 @@ r.get('/visible',
   }
 );
 
+r.get('/badvalidate',
```

Comment:
This works.  Kinda makes me think that we should just have a "test generator" to just run through every data type.